src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamServlet.scala

```diff
@@ -21,13 +21,16 @@ import scala.concurrent.{Await, blocking}
   * @param request the initial http request.
   * @param emitter the emitter to emit data
   */
-class HttpEventSSEHandle(request: HttpServletRequest, emitter: Emitter) extends HttpEventStreamHandle {
+class HttpEventSSEHandle(request: HttpServletRequest, emitter: Emitter, allowHeavyEvents: Boolean) extends HttpEventStreamHandle {
 
   lazy val id: String = UUID.randomUUID().toString
 
   private val subscribedEventTypes = request.getParameterMap.getOrDefault("event_type", Array.empty).toSet
 
-  private val useLightWeightEvents = request.getParameterMap.getOrDefault("plan-format", Array.empty).contains("light")
+  private val useLightWeightEvents = if (allowHeavyEvents)
```

Comment:
so `request.getParameterMap.getOrDefault("plan-format", Array.empty).contains("light") || !allowHeavyEvents` ... ? :)

Reply:
I actually wrote was you suggested first and felt like this was clearer

Reply:
ok, it's a matter of taste, I would prefer the shorter one, but keep it as is if you like that one more :)